streams/src/main/java/org/apache/kafka/streams/processor/internals/StampedRecord.java

```diff
@@ -49,6 +51,20 @@ public Headers headers() {
         return value.headers();
     }
 
+    public ConsumerRecord<byte[], byte[]> rawRecord() {
+        return rawRecord;
+    }
+
+    @Override
+    public boolean equals(final Object other) {
```

Comment:
Why do we add this overload? (Same for `hashCode()` below). -- Seems unnecessary.

Reply:
I see. Thanks. "Interesting" -- or should I rather say "annoying" 😁